### PR TITLE
Update Gradle

### PR DESCRIPTION
**Pull request by karianna**

### CODE REVIEW

File: PCGen-base/gradle/reporting.gradle [hunk 1]
```diff
@@ -33,4 +33,8 @@ jdepend {
 	sourceSets = []
 }
 
+task wrapper(type: Wrapper) {
+	gradleVersion = '4.4.1'
```
Comment (grimreaper) [inline PCGen-base/gradle/reporting.gradle]: this doesn't seem right. We already have this stanza in build.gradle.

Comment (karianna) [reply PCGen-base/gradle/reporting.gradle]: @grimreaper You mean in the build.gradle for this project or the main pcgen (pcgendev) project?

Comment (grimreaper) [reply PCGen-base/gradle/reporting.gradle]: ah, sorry, missed which project this was in. Lets keep them in the same place.

Comment (karianna) [reply PCGen-base/gradle/reporting.gradle]: NP:  So shift this to build.gradle and reporting.gradle will inherit it so to speak?

Comment (grimreaper) [reply PCGen-base/gradle/reporting.gradle]: yesplz

Comment (karianna) [reply PCGen-base/gradle/reporting.gradle]: new PR submitted.

**Pull request merged**